frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -1602,6 +1602,11 @@ export class Arena {
     if (problemMatch && this.problems[problemMatch[1]]) {
       const newRun = problemMatch[2];
       const problem = (this.currentProblem = this.problems[problemMatch[1]]);
+
+      // Set link to edit problem
+      $('#edit-problem-link').text('Editar este problema');
+      $('#edit-problem-link').attr('href', `/problem/${problem.alias}/edit/`);
```

Comment:
Actualmente estamos evitando en lo posible usar `jQuery`, así que este cambio se puede hacer con javascript puro, que te parece algo así:
```suggestion
      document
        .querySelector('#edit-problem-link')
        ?.setAttribute('href', `/problem/${problem.alias}/edit/`);
```
?
Además, como el texto siempre va a ser el mismo, ese se puede agregar desde el archivo `.tpl`. De una vez puedes aprovechar para agregar ese texto en los archivos de traducciones.

Reply:
De acuerdo, ya he hecho el cambio